UI/OptionsWnd.cpp

```diff
@@ -687,12 +773,21 @@ GG::ListBox* OptionsWnd::CreatePage(const std::string& name) {
     return page;
 }
 
-void OptionsWnd::CreateSectionHeader(GG::ListBox* page, int indentation_level, const std::string& name) {
+void OptionsWnd::CreateSectionHeader(
+    GG::ListBox* page, int indentation_level, const std::string& name, const std::string& tooltip)
```

Comment:
In cases where the function name takes up most of the line, sure, but in this case it won't save any space since you use up a full second line and the name is less than that width.